src/Console/TaskCommand.php

```diff
@@ -62,6 +62,13 @@ protected function execute(Input $input, Output $output)
         $stage = $input->hasArgument('stage') ? $input->getArgument('stage') : null;
 
         $tasks = $this->deployer->getScriptManager()->getTasks($this->getName(), $stage);
+
+        if (!$input->getOption('hooks') || $input->getOption('hooks') === 'false') {
+            $tasks = array_filter($tasks, function ($task) {
+                return $this->getName() == $task->getName();
+            });
```

Comment:
@peterjaap I guess you should implement `ScriptManager::setHooksEnabled(true|false)` and use the `$isHooksEnabled` flag in `ScriptManager::getTasks()` method. It will work both for single and group tasks.

Also nice to have tests for that. Check test/src/FunctionsTest.php with pretty same ones.

Peter, could you do that? Do you have a free time?